docs/source/deploy/ledger-topologies.rst

```diff
@@ -69,15 +69,15 @@ Scaling
 
 The main functionalities of a system providing the Ledger API are:
 
-1. serving the API itself (handling the gRPC connections, authenticating users, etc),
+1. handling the gRPC connections,
```

Comment:
Afaik authentication can be quite misleading here. If you think it's important, I suggest "token validation" instead.
Also "serving the API itself" is a bit unclear what it means.

Reply:
Fully agree, it should be authorization. That said, I’m not sure “handling the gRPC connections” is an improvement for a few reasons:

1. There is gRPC in a bunch of places, e.g., Canton and kvutils also uses it between participants. We should be specific that we are talking about the gRPC connections to the ledger API.
2. “handling gRPC connections” sounds like a very low-level thing and I wouldn’t expect authorization to be included here but based on the previous wording we deliberately do want to include this.

So maybe just replacing authentication by authorization is sufficient?

Reply:
Could work, but please let me remove the "etc" too. The text above suggest that there may be other tasks ("main functionalities"), so "etc" is not necessary.

(Note: I'm also slightly against the text "serving the API itself", because to me it would mean everything, including the other items as well. The explanation clarifies it a bit, but that mysterious "etc" takes away the clarification, for me at least :) )

So how about:
```suggestion
1. serving the API itself (handling the gRPC connections, authorizing users),
```